### PR TITLE
Refactor/wave bye jquery

### PR DESCRIPTION
**Pull request by austinezell**
## Description

This pull request addresses the issue of using jQuery in 2019. 

These are the changes included:

- :white_check_mark: Die jQuery
## Tests

- [ X] All tests are _definitely_ passing

### CODE REVIEW

**Comment by andybeers** [comment]
oh my god

File: addon/components/rad-popover/content.js [hunk 1]
```diff
@@ -142,30 +141,24 @@ export default Component.extend({
...
 
     /*
      * If the box is centered, it will center itself back off of the page when we
      * subtract the necessary width from the component width. In these cases, we
      * will need to subtract twice the necessary width. The box is only ever centered
      * when position does not contain `-left`/`-right`.
      */
-    const boxIsCentered =
-      this.get('position') === 'top' || this.get('position') === 'bottom'
+    const boxIsCentered = this.position === 'top' || this.position === 'bottom'
```
Comment (dchasepdx) [inline addon/components/rad-popover/content.js]: Can you use `position` that was destructed above?

File: addon/utils/elements.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+    parentNode = parentNode.parentNode
+  }
+  return false
+}
+
+/**
+ * Determine the outerHeight including margins of an element
+ * @param {Element} el
+ */
```
Comment (andybeers) [inline addon/utils/elements.js]: can you add `@returns` to these utils :)

**Comment by dchasepdx** [comment]
So, can we remove the actual `@ember/jquery` dependency?

**Comment by austinezell** [comment]
@dchasepdx 
> So, can we remove the actual `@ember/jquery` dependency?

No. It is still being used in demo app.

File: addon/components/rad-popover/content.js (hunk 1 above)
Comment (austinezell) [reply addon/components/rad-popover/content.js]: Done

**Pull request merged**